examples/duel/.gitignore

```diff
@@ -1 +1,3 @@
 target/
+node_modules/
```

Comment:
What is "improper" about this one? It excludes the dirs that I don't want included.

The dependencies needed to build the app are included in the webapp tree. See how package.json copies them after `npm install`.